src/main/java/com/actiontech/dble/backend/delyDetection/DelayDetectionSqlJob.java

```diff
@@ -29,24 +29,24 @@ public class DelayDetectionSqlJob implements ResponseHandler {
     private long versionVal;
     private AtomicBoolean finished = new AtomicBoolean(false);
     private LocalDateTime responseTime;
-    private long keepAlive;
+    private long keepAlive = 60000;
 
     public DelayDetectionSqlJob(DelayDetection delayDetection, OneRawSQLQueryResultHandler jobHandler) {
         this.delayDetection = delayDetection;
         this.jobHandler = jobHandler;
         int delayPeriodMillis = delayDetection.getSource().getDbGroupConfig().getDelayPeriodMillis();
-        this.keepAlive = delayDetection.getSource().getDbGroupConfig().getKeepAlive() + delayPeriodMillis;
+        this.keepAlive += delayPeriodMillis;
```

Comment:
also need to modify equalsForDelayDetection method, accordingly.

Reply:
ok